core/src/main/resources/application.conf

```diff
@@ -49,28 +53,37 @@ rvi {
 }
 
 resolver = {
-  baseUri = "http://localhost:8081"
-  baseUri = ${?RESOLVER_API_URI}
+  host = "localhost"
+  host = ${?SOTA_RESOLVER_HOST}
+  port = 8081
+  port = ${?SOTA_RESOLVER_PORT}
+  baseUri = "http://"${resolver.host}":"${resolver.port}
   packagesUri = ${resolver.baseUri}"/api/v1/resolver/packages"
   resolveUri = ${resolver.baseUri}"/api/v1/resolver/resolve"
   vehiclesUri = ${resolver.baseUri}"/api/v1/resolver/devices"
 }
 
 device_registry = {
-  baseUri = "http://localhost:8083"
-  baseUri = ${?DEVICE_REGISTRY_API_URI}
-  devicesUri = "/api/v1/devices"
-  deviceGroupsUri = "/api/v1/device_groups"
```

Comment:
It's a `Uri`. It's gets parsed into a `Uri` object. Only the `getPath` accessor on that object is called, so it's used only as a path in the code. I didn't bother changing the code since it still works.